api/db/seeds.rb

```diff
@@ -3,65 +3,72 @@
 # Seed data is contained in the json files under seed folder. This file serves as the script of 
 # pasing json file and create tables from the parsed data. The data can then be loaded with the rails db:seed command (or created alongside the
 # database with db:setup). For testing purpose, use db:reset to reload all the table.
+include SeedsHandler
 
-seed_data_dir = Rails.root.join('db', 'seed')
+chaining = [
+    {
+        get: '/sessions',
+        create: '/sessions',
+        unique: [:name]
```

Comment:
`unique` is a confusing name. What do you think of `index_on`?

Reply:
Fixed.